ksqldb-cli/pom.xml

```diff
@@ -154,6 +154,13 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Those are the original changes from #7808 (included for not to make the build pass)